cachito/nexus_scripts/configure_nexus.groovy

```diff
@@ -0,0 +1,206 @@
+// This script configures Nexus to be ready for Cachito and is idempotent
+import com.google.common.collect.Sets
+import groovy.json.JsonSlurper
+import org.sonatype.nexus.repository.config.Configuration
+import org.sonatype.nexus.repository.storage.WritePolicy
+import org.sonatype.nexus.security.authz.AuthorizationManager
+import org.sonatype.nexus.security.role.NoSuchRoleException
+import org.sonatype.nexus.security.role.Role
+import static org.sonatype.nexus.security.user.UserManager.DEFAULT_SOURCE
+import org.sonatype.nexus.security.user.UserNotFoundException
+
+
+def createBlobStore(String blobStoreName) {
+    if (!blobStore.blobStoreManager.get(blobStoreName)) {
+        String blobStorePath = blobStoreName
```

Comment:
This looks odd. Why do we need to make a reference copy to `blobStoreName`? Are both variables really supposed to have the same value? If so, let's comment why.

Reply:
This is intentional. I will add a comment.

Reply:
@lcarva the extra variable is not necessary, but it makes it clear what the input is to the method.